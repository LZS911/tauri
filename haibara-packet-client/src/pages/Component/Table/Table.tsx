import { Button } from 'antd';

const Table: React.FC = () => {
  return <Button type="primary">Table</Button>;
};
export default Table;
