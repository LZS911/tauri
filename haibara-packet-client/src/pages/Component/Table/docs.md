# Generate Table page by gl-cli
