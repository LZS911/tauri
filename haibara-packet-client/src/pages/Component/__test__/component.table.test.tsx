import Table from '../Table';
import { render } from '@testing-library/react';
describe('test Component', () => {
  it('should render Table to match snapshot', () => {
    const { container } = render(<Table />);
    expect(container).toMatchSnapshot();
  });
});
