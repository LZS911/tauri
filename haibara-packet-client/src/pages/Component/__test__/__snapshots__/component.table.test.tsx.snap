// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Component should render Table to match snapshot 1`] = `
<div>
  <div>
    Table
  </div>
</div>
`;
